subsys/nfc/lib/platform_internal_thread.c

```diff
@@ -185,28 +213,35 @@ static void schedule_callback(void)
 #endif /* CONFIG_NFC_OWN_THREAD */
 }
 
-#ifdef CONFIG_NFC_LOW_LATENCY_IRQ
+#ifdef CONFIG_NFC_ZERO_LATENCY_IRQ
 ISR_DIRECT_DECLARE(nfc_swi_handler)
 {
+	atomic_val_t tmp;
```

Comment:
I actually can't do it, because other client could have altered the clock status and if nfc repeated for example clock start request, the old request would be the same as current and clock would not be started.